README.md

```diff
@@ -614,7 +641,8 @@ that have not been customized via a node label (more on this later).
 
 #####  Single Config File Example
 As an example, create a valid config file on your local filesystem, such as the following:
-```
+
+```yaml
```

Comment:
why is this not `shell` when the example on line 696 is?

Reply:
Standardized.